installation/resources/crds/eventing/customeresourcedefinition.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
these crds are already here: https://github.com/kyma-project/kyma/blob/main/installation/resources/crds/eventing/natsclusters.nats.crd.yaml
https://github.com/kyma-project/kyma/blob/main/installation/resources/crds/eventing/natsserviceroles.nats.crd.yaml

Reply:
You're right. Fixed.